package.json

```diff
@@ -1,12 +1,9 @@
 {
   "name": "@reason-react-native/react-navigation",
-  "version": "3.11.0-alpha.0",
+  "version": "4.0.0-alpha.0",
   "publishConfig": {
     "access": "public"
   },
-  "peerDependencies": {
-    "react-navigation": "^3.11.0"
-  },
```

Comment:
My idea of placing peered next to the version is to ensure we follow our versioning "rule" for the binding. Any thoughts about keeping peerDep around here?